x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -63,55 +82,186 @@ export const installTransform = async (
   const transformPaths = paths.filter((path) => isTransform(path));
   let installedTransforms: EsAssetReference[] = [];
   if (transformPaths.length > 0) {
-    const transformRefs = transformPaths.reduce<EsAssetReference[]>((acc, path) => {
-      acc.push({
-        id: getTransformNameForInstallation(installablePackage, path, installNameSuffix),
-        type: ElasticsearchAssetType.transform,
-      });
+    const transformsSpecifications = new Map();
+    const destinationIndexTemplates: DestinationIndexTemplateInstallation[] = [];
+    const indices = [];
+    const transforms: TransformInstallation[] = [];
 
-      return acc;
-    }, []);
+    transformPaths.forEach((path: string) => {
+      const { transformModuleId, fileName } = getTransformFolderAndFileNames(
+        installablePackage,
+        path
+      );
 
-    // get and save transform refs before installing transforms
-    esReferences = await updateEsAssetReferences(
-      savedObjectsClient,
-      installablePackage.name,
-      esReferences,
-      {
-        assetsToAdd: transformRefs,
+      // Since there can be multiple assets per transform job definition
```

Comment:
We don't normally use the term `job` for transforms, so `job` should be removed from this line and the one below.

Reply:
Removed